src/styles.css

```diff
@@ -0,0 +1,186 @@
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font: inherit;
+  font-size: 100%;
+  vertical-align: baseline;
+}
+
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+
+ol,
+ul {
+  list-style: none;
+}
+
+blockquote,
+q {
+  quotes: none;
+}
+
+blockquote::before,
+blockquote::after,
+q::before,
+q::after {
+  content: '';
+  content: none;
+}
+
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+
+body {
+  width: 417px;
+  height: 718px;
+}
+
+header {
+  width: 288px;
+  padding: 0  20px;
+  display: flex;
+  justify-content: space-between;
+  height: 100px;
+}
+
+header .logo {
+  position: relative;
+  top: 39px;
+  width: 109px;
+  height: 20px;
+  font-family: Poppins, serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 28px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: rgb(96, 112, 255);
+  text-decoration: none;
+}
+
+header .enabled {
+  height: 20px;
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  position: absolute;
+  top: 29px;
+  border-radius: 8px;
+}
+
+section {
+  background-image: url("../assets/header-shapes\ mobile.png");
+  background-repeat: not-repeat;
+  width: 100%;
+}
```

Comment:
- [ ] [REQUIRED] Good job so far! Please implement `bottom` and `right` of your project like [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=rVw9jLSLBaZEWhxj-0) because your bottom and left part of your project have a lot of unneccery space 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/219718106-69f67054-b890-4b95-91ba-0fa418015d2e.png)